packages/qemu-system-x86-64-headless/build.sh

```diff
@@ -107,4 +112,11 @@ termux_step_post_make_install() {
 			"${TERMUX_PREFIX}"/share/man/man1/qemu.1 \
 			"${TERMUX_PREFIX}"/share/man/man1/qemu-system-${i}.1
 	done
+	# patchelf qemu binary to correct needed library of spice
+	local b
+        for b in aarch64 arm i386 riscv32 riscv64 x86_64; do
+		patchelf --replace-needed \
+			libspice-server.so.1 libspice-server.so \
```

Comment:
Wouldn't it be easier to create a symlink for the library?

Reply:
Extremely sorry,
I made mistake. Wil fix it 😉